src/managing-workflow/platform-logging.md

```diff
@@ -10,14 +10,7 @@ The logging platform is made up of 2 components - [Fluentd](https://github.com/d
 Logger acts like a syslog server and receives all log messages that are occurring on the cluster. It then filters this data to only Deis deploy applications and stores those log messages in a ring buffer where they can be fetched via the Deis CLI.
 
 ## Installation
-Installation of the logging components is separate from the installation of the Deis platform because of the requirement of Daemon Sets. You will need to follow these steps to get the logging components running on your kubernetes cluster.
-
-```
-$ helm fetch deis/deis-logger-beta1-test
-$ helm install deis-logger-beta1-test
-```
-
-You will then need to watch the components come up and verify they are in a running state by executing the following command:
+With the release of workflow-beta2 the logging system is part of the main installation of deis. You will then need to watch the components come up and verify they are in a running state by executing the following command:
```

Comment:
Instead of "deis" we should refer to "Workflow" or "Deis Workflow" consistently:
> With the release of workflow-beta2, the logging system is part of every Workflow installation.